library/codeop.po

```diff
@@ -30,32 +30,44 @@ msgid ""
 "include such a loop in your program you probably want to use the :mod:`code` "
 "module instead."
 msgstr ""
+"Le module :mod:`codeop` fournit des outils permettant d'émuler une boucle de "
+"lecture-évaluation-affichage (en anglais *read-eval-print-loop* ou REPL), "
+"comme dans le module :mod:`code`.  Par conséquent, ce module n'est pas "
+"destiné à être utilisé directement ; pour inclure un REPL dans un programme, "
+"il est préférable d'utiliser le module :mod:`code`."
 
 #: ../Doc/library/codeop.rst:20
 msgid "There are two parts to this job:"
-msgstr ""
+msgstr "Cette tâche se divise en deux parties :"
 
 #: ../Doc/library/codeop.rst:22
 msgid ""
 "Being able to tell if a line of input completes a Python  statement: in "
 "short, telling whether to print '``>>>``' or '``...``' next."
 msgstr ""
+"Pouvoir affirmer qu'une ligne d'entrée est une instruction complète, ou "
+"achève une instruction : en bref, savoir s'il faut afficher « ``>>>`` » ou "
+"« ``...`` » à sa suite."
 
 #: ../Doc/library/codeop.rst:25
 msgid ""
 "Remembering which future statements the user has entered, so  subsequent "
 "input can be compiled with these in effect."
 msgstr ""
+"Conserver les instructions déjà entrées par l'utilisateur, afin que les "
+"entrées suivantes puissent êtres compilées avec elles."
 
 #: ../Doc/library/codeop.rst:28
 msgid ""
 "The :mod:`codeop` module provides a way of doing each of these things, and a "
 "way of doing them both."
 msgstr ""
+"Le module :mod:`codeop` fournit un moyen d'effectuer ces deux parties, "
+"individuellement ou simultanément."
 
 #: ../Doc/library/codeop.rst:31
 msgid "To do just the former:"
-msgstr ""
+msgstr "Pour les effectuer simultanément :"
```

Comment:
"the former" c'est le premier, pourquoi traduire par "les deux" ?

Reply:
Oui 'former' pas 'latter'.

Reply:
Oui "former" pas "les deux simultanément" ?